src/utils/download.ts

```diff
@@ -0,0 +1,53 @@
+import * as fs from 'fs';
+import path from 'path';
+import axios from 'axios';
+
+type DownloadProgressCallback = (downloadProgress: number) => void;
+type DownloadResult = { pathToFile: string; contentType: string };
+
+/**
+ * Downloads file from remote HTTP[S] host and puts its contents to the
+ * specified location.
+ * @param url URL of the file to download.
+ * @param destinationPath Path to the destination file.
+ * @param destinationFileName The file name.
+ */
+export async function download(
```

Comment:
nit: can we add empty lines to separate the blocks and increase the legibility here?